graph/src/components/versions/registry.rs

```diff
@@ -0,0 +1,87 @@
+use crate::prelude::FeatureFlag;
+use lazy_static::lazy_static;
+use semver::{Version, VersionReq};
+use std::collections::HashMap;
+
+lazy_static! {
+    pub static ref VERSIONS: HashMap<Version, Vec<FeatureFlag>> = {
+        let mut supported_versions: Vec<(Version, Vec<FeatureFlag>)> = vec![
+            // baseline version
+            (Version::new(1, 0, 0), vec![]),
+            // Versions with feature flags
+            (Version::new(1, 1, 0), vec![FeatureFlag::BasicOrdering])
+        ];
+
+        let mut map = HashMap::new();
+
+        // Sort version by major, minor, patch, from higher to lower.
+        supported_versions.sort_by(|a, b| b.0.partial_cmp(&a.0).unwrap());
```

Comment:
Is this sort necessary? They all go into a `HashMap` afterwards, so doesn't seem like it's needed.

Reply:
I did a refactor, now only the vector is sorted